libs/application/templates/new-primary-school/src/lib/NewPrimarySchoolTemplate.ts

```diff
@@ -491,13 +491,21 @@ const NewPrimarySchoolTemplate: ApplicationTemplate<
         )
 
         if (
-          selectedSchoolSubType !== OrganizationSubType.INTERNATIONAL_SCHOOL
+          selectedSchoolSubType !== OrganizationSubType.INTERNATIONAL_SCHOOL &&
+          selectedSchoolSubType !==
+            OrganizationSubType.SPECIAL_EDUCATION_BEHAVIOR_SCHOOL &&
+          selectedSchoolSubType !==
+            OrganizationSubType.SPECIAL_EDUCATION_BEHAVIOR_DEPARTMENT
         ) {
           unset(application.answers, 'startingSchool.temporaryStay')
           unset(application.answers, 'startingSchool.expectedEndDate')
         }
         if (
-          selectedSchoolSubType === OrganizationSubType.INTERNATIONAL_SCHOOL &&
+          (selectedSchoolSubType === OrganizationSubType.INTERNATIONAL_SCHOOL ||
+            selectedSchoolSubType ===
+              OrganizationSubType.SPECIAL_EDUCATION_BEHAVIOR_SCHOOL ||
+            selectedSchoolSubType ===
+              OrganizationSubType.SPECIAL_EDUCATION_BEHAVIOR_DEPARTMENT) &&
```

Comment:
Shouldn't we also use `subTypeWithExpectedEndDate()` here? 🤔 

```suggestion
          subTypeWithExpectedEndDate(selectedSchoolSubType) &&
```